client/peerset/src/lib.rs

```diff
@@ -527,11 +603,6 @@ impl Peerset {
 	pub fn num_discovered_peers(&self) -> usize {
 		self.data.peers().len()
 	}
-
-	/// Returns priority group by id.
-	pub fn get_priority_group(&self, group_id: &str) -> Option<HashSet<PeerId>> {
```

Comment:
This should be exposed to RPC really. I'd keep it, but rename to `priority_group`